src/js/edu-benefits/1990e/config/form.js

```diff
@@ -31,7 +48,36 @@ const formConfig = {
       pages: {
         applicantInformation: {
           path: 'applicant-information',
-          title: 'Applicant information'
+          title: 'Applicant information',
+          initialData: {},
+          uiSchema: {
+            applicantFullName: fullName.uiSchema,
+            applicantSocialSecurityNumber: ssn.uiSchema,
+            applicantDateOfBirth: currentOrPastDate.uiSchema('Applicant date of birth'),
```

Comment:
@goldenmeanie Another question: is it true that relatives of veterans applying for veterans must be 17 years of age? I poked around in our docs and (many) online resources, including vonapp and paper form and can't see that restriction. However, I did see somewhere that once the relative reaches 26 years of age, s/he is no longer eligible. Need to know so we can set up the validations properly.